main.nf

```diff
@@ -1,17 +1,21 @@
 #!/usr/bin/env nextflow
 nextflow.enable.dsl=2 
 
-process sayHello {
+params.list_file = 'list.txt'
+
+process countLines {
```

Comment:
When I ran `main.nf`, there was a warning about `countLines` overriding a built-in operator. You should change the name of the process to avoid this. 

```console
❯ nextflow run main.nf --list_file list.txt 

Launching `main.nf` [small_goldstine] - revision: c53926ba83
WARN: Process with name 'countLines' overrides a built-in operator with the same name
executor >  local (1)
[23/f819e4] process > countLines (1) [100%] 1 of 1 ✔
File: main.nf has total of 21 lines 
File: README.md has total of 4 lines 
File: https://raw.githubusercontent.com/nextflow-io/nextflow/master/README.md has total of 277 lines
```

Reply:
@BrunoGrandePhD I've updated the file. I have no idea why GitHub didn't let me `Re-request` a review from you (I clicked the button but nothing happened so I'm just gonna tag you in each comment, lol